source/Application/Controller/Admin/LanguageMain.php

```diff
@@ -507,11 +501,7 @@ protected function _validateInput()
      */
     protected function checkAbbreviationAllowedCharacters($abbreviation)
     {
-        $return = false;
-        if (preg_match('/^[a-zA-Z0-9_]*$/', $abbreviation)) {
-            $return = true;
-        }
-        return $return;
+        return preg_match('/^[a-zA-Z0-9_]*$/', $abbreviation);
```

Comment:
Copy paste from php.net: "preg_match() returns 1 if the pattern matches given subject, 0 if it does not, or FALSE if an error occurred."
Method should return only `true` or `false`.